spec/apollo-federation/service_field_spec.rb

```diff
@@ -408,75 +406,64 @@ def self.visible?(context)
       )
     end
 
-    context 'with a filter' do
+    context 'with context in schema generation' do
```

Comment:
This spec is testing that schema generation (`#federation_sdl`) respects a passed in context. But we tested this indirectly by using a filter (`PermissionWhitelist`). I think using a filter class like this is deprecated. 

With the above, just decided to rewrite the spec to make it more direct.